test/functional/fixtures/es_archiver/logstash_functional/mappings.json

```diff
@@ -1088,4 +1097,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Fixed but it can't tell because you couldn't comment on the line and this is too far away :(